releases.md

```diff
@@ -40,6 +40,13 @@ Further documentation available:
 
 ## Releases
 
+### v0.26 (LTS)
```

Comment:
I forgot how we decide LTS for triggers :thinking: is it based on Pipelines LTS ?

Reply:
Now, it's all LTS. Earlier, we were doing more releases, so release align with Pipelines LTS was LTS.